src/parsers/micro-rdfa-parser.js

```diff
@@ -1,6 +1,7 @@
 const htmlparser = require("htmlparser2");
 
 function getPropValue(tagName, attribs, TYPE, PROP) {
+  if (tagName === "meta" && attribs.itemtype) return null;
```

Comment:
I don't think you need this anymore because you delete the itemtype before calling this method.